gravity.sh

```diff
@@ -85,8 +82,6 @@ fi
 # Generate new sqlite3 file from schema template
 generate_gravity_database() {
   sqlite3 "${gravityDBfile}" < "${gravityDBschema}"
-  chown ${DATABASE_USER}:${DATABASE_GROUP} "${piholeDir}" "${gravityDBfile}"
-  chmod ${DATABASE_PERMISSIONS} "${piholeDir}" "${gravityDBfile}"
```

Comment:
Why are the permissions and user ownership removed?

Reply:
`sqlite3` creates the database with `rw-r--r--`, owned by `root:root`. As only `gravity.sh` and `lists.sh` need write access (they are `root`), this is sufficient. The ownership code slipped in there accidentally and was part of trying to modify the database directly from the web interface. I think it is safe to say that the fewer users have write permissions on this database, the better.

We could still revisit this in a follow-up PR.

Reply:
These permissions will prevent the (current) web interface and the new API from accessing the database. The web interface directly interacts with the database when reading.

Reply:
Do we want to allow anyone to read the database? This will make it 644.